RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -119,6 +144,11 @@ void PixelThresholdClusterizer::clusterizeDetUnitT( const T & input,
   
   detid_ = input.detId();
   
+  // Set separate cluster threshold for L1 (needed for phase1)
+  auto clusterThreshold = theClusterThreshold;
+  int layer = (DetId(detid_).subdetId()==1) ? PXBDetId(detid_).layer() : 0;
```

Comment:
Uh, you are absolutely right, the bit coding is only same for phase 0. We must use TrackerTopology indeed.